### PR TITLE
Ergo backlog

### PR DESCRIPTION
**Pull request by glasgowm148**

### CODE REVIEW

File: data/ecosystems/e/ergo-oracle.md
Comment (eherrerosj) [inline data/ecosystems/e/ergo-oracle.md:1]: can you remove this empty file?

File: data/ecosystems/s/swamp-audio.toml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Ecosystem Level Information
-title = "Swamp Audio"
+title = "TremendouslyHighFrequency"
 
 sub_ecosystems = []
 
-github_organizations = []
+github_organizations = ["https://github.com/TremendouslyHighFrequency"]
```
Comment (eherrerosj) [inline data/ecosystems/s/swamp-audio.toml:6]: This is a user URL, not an Organization URL, can you remove?

File: data/ecosystems/m/minotaur.toml [hunk 2]
```diff
@@ -4,113 +4,18 @@ title = "Minotaur"
...
-url = "https://github.com/rosen-bridge/coming-soon"
-
-[[repo]]
-url = "https://github.com/rosen-bridge/contract"
-
-[[repo]]
-url = "https://github.com/rosen-bridge/contract-test"
-
-[[repo]]
-url = "https://github.com/rosen-bridge/guard-service"
```
Comment (eherrerosj) [inline data/ecosystems/m/minotaur.toml:65]: can you move these repos to rosen-bridge.toml?

File: data/ecosystems/m/minotaur.toml [hunk 3]
```diff
@@ -4,113 +4,18 @@ title = "Minotaur"
...
-url = "https://github.com/ErgoGravity/gateway-proxy"
-
-[[repo]]
-url = "https://github.com/ErgoGravity/gravity-core"
-
-[[repo]]
-url = "https://github.com/ErgoGravity/gravity-node-data-extractor"
-
-[[repo]]
-url = "https://github.com/ErgoGravity/startup-script"
```
Comment (eherrerosj) [inline data/ecosystems/m/minotaur.toml:32]: can you move these to a new eco?

**Review by eherrerosj — COMMENTED** [review]
left a few comments

**Comment by jubos** [comment]
@glasgowm148, this is getting close.  Can you rebase the changes and apply them cleanly against the master branch and then I think it's good for merge.

**Review by jubos — CHANGES_REQUESTED** [review]
Let's rebase and apply cleanly to the master branch since there have been a ton of changes since this was originally created.

File: data/ecosystems/m/minotaur.toml (hunk 3 above)
Comment (glasgowm148) [reply data/ecosystems/m/minotaur.toml:32]: These were for a bridge that pre-dates Rosen, built by the same team. The bridge was abandoned due to the wormhole hack so this makes the most sense to include to show the development history of the team (as inevitably some of the logic/understanding gained from these were used in Rosen)

**Comment by glasgowm148** [comment]
What do you mean @jubos ? It's already up to date with origin/master

**Comment by glasgowm148** [comment]
Seems fine for me? 

Screenshot 2024-01-09 at 11 38 07

**Comment by glasgowm148** [comment]
cherry-picked my commits, hows that?

**Comment by jubos** [comment]
What I would do is squash all of your commits down into one commit that makes all of the Ergo related changes on your branch.

You can create a new branch based on the very latest master ([792f8de](https://github.com/electric-capital/crypto-ecosystems/commit/792f8de9c8ebe8d0e93a335133d850b3c5d63e29) at the time of this comment).

Then do 
```git cherry-pick <your squashed commit>```

Fix the conflicts if there are any and then either force push to this PR or create a new clean PR.

**Comment by glasgowm148** [comment]
Why though you can see all the files changed it's not a complicated commits it's just some .toml files. 

Becoming a bit of a joke now - please merge!

**Pull request merged**